external-api/src/external-api/bank-holidays.test.ts

```diff
@@ -0,0 +1,18 @@
+import { isBefore } from 'date-fns';
+
+describe('bank-holidays', () => {
+  /**
+   * If this test fails it means it is time to update the bank holiday backup data.
+   * The backup should be retrieved from https://www.gov.uk/bank-holidays.json.
+   *
+   * After updating the backup, the deadline in the test should be extended
+   * to be 6 months before the date of the final bank holiday in the backup.
+   */
+  it('should have up to date backup data for bank holidays', () => {
+    const deadlineForUpdatingBackup = new Date('2026-07-01');
```

Comment:
This is a year in the future compared to the ticket AC because I updated the back up data before realising the ticket didn't actually require me to do that but then I thought might as well since I had done it so extended the deadline also. 

@ChristianMcc is this the correct periodicity for updating?